lib/iban-tools/iban_rules.rb

```diff
@@ -15,15 +15,23 @@ def [](key)
     end
 
     def self.defaults
-      load_from_string( File.read(File.dirname(__FILE__) + "/rules.yml") )
+      load_from_string
     end
 
-    def self.load_from_string( string )
-      rule_hash = YAML.load(string)
-      rule_hash.each do |country_code, specs|
+    def self.load_from_string
+      hash_copy = rule_hash
```

Comment:
This doesn’t actually create a copy. 🤔

What seems to be the issue you are trying to fix :?

Reply:
lol, I really don't know after 7 years...

Reply:
Haha, I think it can be closed then 😅

Reply:
y